subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3564,6 +3565,26 @@ static void le_adv_ext_coded_report(struct pdu_data *pdu_data,
 {
 	le_adv_ext_report(pdu_data, node_rx, buf, BIT(2));
 }
+
+static void le_adv_ext_terminate(struct pdu_data *pdu_data,
+				    struct node_rx_pdu *node_rx,
+				    struct net_buf *buf)
+{
+	struct bt_hci_evt_le_adv_set_terminated *sep;
+
+	if (!(event_mask & BT_EVT_MASK_LE_META_EVENT) ||
+	    !(le_event_mask & BT_EVT_MASK_LE_ADV_SET_TERMINATED)) {
+		return;
+	}
+
+	sep = meta_evt(buf, BT_HCI_EVT_LE_ADV_SET_TERMINATED, sizeof(*sep));
+	sep->num_completed_ext_adv_evts =
+		(uint8_t)((uint32_t)node_rx->hdr.rx_ftr.extra & 0xff);
```

Comment:
I would prefer if rx_ftr would have explicit fields to carry the respective information instead of using the generic "extra" field. If space is of the essence, use a union.